docs/_includes/tests/filter-menu-button.html

```diff
@@ -0,0 +1,650 @@
+<h2>Collapsed</h2>
+<div class="demo">
+    <div class="demo__inner">
+        <div class="filter-group">
+            <span class="filter-menu-button">
+                <button type="button" class="filter-menu-button__button">
+                    <span class="filter-menu-button__button-cell">
+                        <span class="filter-menu-button__button-text">Filter Menu Button</span>
+                        <svg class="icon icon--chevron-down" focusable="false" height="12" width="12">
+                            <use xlink:href="#icon-chevron-down"></use>
+                        </svg>
+                    </span>
+                </button>
+                <div class="filter-menu-button__menu">
+                    <div class="filter-menu-button__items" role="menu">
+                        <div class="filter-menu-button__item" role="menuitemcheckbox" aria-checked="false">
+                            <span class="filter-menu-button__status--inline">
```

Comment:
@seangates Sorry, but I just spotted this `filter-menu-button__status--inline`. This is actually invalid BEM. You can't have a modifier class without a base class.

Reply:
A modifier is invalid BEM?

Reply:
Yup. A modifier is supposed to modify a base class. In this case, `filter-menu-button__status`

Reply:
I was wondering about that when I saw it too, since I'd never seen it before.  But according to the BEM docs modifiers are `Flags on blocks or elements.`
So `filter-menu-button__status` would be the element and `--inline` would be the modifier, right?

Reply:
But I'm assuming there is a `filter-menu-button__status` element without the modifier.  So I guess basically there is no need for the modifier if there is no "unmodified" version of the element.

Reply:
If you only have one class, what other class styles are you modifying in the cascade?

Reply:
Right, so it would have to be `filter-menu-button__status filter-menu-button__status--inline` but there's no need for the modifier if it isn't actually modifying anything from `filter-menu-button__status`.  Got it.